src/wallet/wallet.cpp

```diff
@@ -1954,6 +1954,39 @@ SpendableInputs CWallet::FindSpendableInputs(
         }
     }
 
+    // for Orchard, we select both the internal and external IVKs.
+    auto orchardIvks = std::visit(match {
+        [&](const libzcash::UnifiedFullViewingKey& ufvk) -> std::vector<OrchardIncomingViewingKey> {
+            auto fvk = ufvk.GetOrchardKey();
+            if (fvk.has_value()) {
+                return {fvk->ToIncomingViewingKey(), fvk->ToInternalIncomingViewingKey()};
+            }
+            return {};
+        },
+        [&](const AccountZTXOPattern& acct) -> std::vector<OrchardIncomingViewingKey> {
+            auto ufvk = GetUnifiedFullViewingKeyByAccount(acct.GetAccountId());
+            if (ufvk.has_value()) {
+                auto fvk = ufvk->GetOrchardKey();
+                if (fvk.has_value()) {
+                    return {fvk->ToIncomingViewingKey(), fvk->ToInternalIncomingViewingKey()};
+                }
+            }
+            return {};
+        },
+        [&](const auto& addr) -> std::vector<OrchardIncomingViewingKey> { return {}; }
+    }, selector.GetPattern());
+
+    for (const auto& ivk : orchardIvks) {
+        // TODO ORCHARD: Allow the minimum number of confirmations
+        // to be specified
```

Comment:
Is there an issue open for this?

Reply:
https://github.com/zcash/zcash/issues/5599